### PR TITLE
Fix JS and CSS issues in master/3.2

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

**Comment by rxu** [comment]
:+1:

File: styles/prosilver/template/ideas.js [hunk 1]
```diff
@@ -205,9 +207,11 @@
 
 			$.get(url, {ticket: value && info[1]}, function(res) {
 				if (res) {
-					$obj.ticketLink.text(value)
-						.attr('href', 'http://tracker.phpbb.com/browse/' + value)
-						.show();
+					if (value) {
+						$obj.ticketLink.text(value)
+							.attr('href', 'http://tracker.phpbb.com/browse/' + value)
```
Comment (callumacrae) [inline styles/prosilver/template/ideas.js]: https?

File: styles/prosilver/theme/ideas.css [hunk 2]
```diff
@@ -93,3 +93,5 @@ dd.topics {
 .voteslist ul, .voteslist ul li { list-style: none; }
 
 .view-all, .view-all:hover { color: #536482; }
+
+#duplicatelink { margin: 0 5px 0 0; }
```
Comment (callumacrae) [inline styles/prosilver/theme/ideas.css]: Please don't use IDs in CSS unless you really have to :(

Comment (VSEphpbb) [inline styles/prosilver/theme/ideas.css]: Well I'd rather not create a whole class just to tweak this one element a little. FWIW, there are already other ID selectors in this CSS file anyways ;)

File: styles/prosilver/template/ideas.js (hunk 1 above)
Comment (VSEphpbb) [inline styles/prosilver/template/ideas.js]: ah, good catch.

File: styles/prosilver/theme/ideas.css (hunk 2 above)
Comment (callumacrae) [inline styles/prosilver/theme/ideas.css]: That doesn't make it okay!!

Comment (VSEphpbb) [inline styles/prosilver/theme/ideas.css]: Feel free to send a PR converting all IDs to classes :wink:

**Comment by VSEphpbb** [comment]
@michaelcullum :smile:

**Pull request merged**